klipper_config/lights.cfg

```diff
@@ -1,2 +1,10 @@
 [output_pin caselights]
 pin: z:P2.5
+
+[gcode_macro caselights_on]
+gcode:
+    set_pin pin=caselights value = 1
+
+[gcode_macro caselights_off]
+gcode:
+    set_pin pin=caselights value = 0
```

Comment:
I think we should remove the spaces here so it is consistent with other places we are using `set_pin`.  `value=0` instead of `value = 0`